### ISSUE TITLE
Cannot compile 

### ISSUE THREAD

**Issue by Yeyvo** [aCLr/rust-tdlib#15]
Hello i was trying to use the lib (version 0.4.0) but here's what happens when i try to compile the example bellow : 

```
use rust_tdlib::client::Worker;
use rust_tdlib::types::{GetMe, TdlibParameters};

#[tokio::main]
async fn main() {
   let mut worker = Worker::builder().build().unwrap();
    let waiter = worker.start();
    let tdlib_params = TdlibParameters::builder().api_id(env!("id").parse::<i32>().unwrap()).api_hash(env!("HASH")).build();
    let client = rust_tdlib::client::Client::builder().with_tdlib_parameters(tdlib_params).build();
    let (client_state, client) = worker.bind_client(client1).await.unwrap();
    let me = client.get_me(GetMe::builder().build()).await.unwrap();
    println!("{:?}", me);
}
```

i use rustc 1.62.1 (e092d0b6b 2022-07-16)

### Error log
```C:/Users/****/.cargo/bin/cargo.exe build --color=always --message-format=json-diagnostic-rendered-ansi --package c2-POC-agent --bin c2-POC-agent
   Compiling proc-macro2 v1.0.42
   Compiling unicode-ident v1.0.2
   Compiling quote v1.0.20
   Compiling syn v1.0.98
   Compiling autocfg v1.1.0
   Compiling futures-core v0.3.21
   Compiling futures-channel v0.3.21
   Compiling futures-task v0.3.21
   Compiling memchr v2.5.0
   Compiling pin-project-lite v0.2.9
   Compiling futures-util v0.3.21
   Compiling futures-sink v0.3.21
   Compiling cfg-if v1.0.0
   Compiling pin-utils v0.1.0
   Compiling futures-io v0.3.21
   Compiling serde v1.0.140
   Compiling async-trait v0.1.56
   Compiling log v0.4.17
   Compiling serde_json v1.0.82
   Compiling serde_derive v1.0.140
   Compiling ryu v1.0.10
   Compiling itoa v1.0.2
   Compiling lazy_static v1.4.0
   Compiling getrandom v0.2.7
   Compiling uuid v0.8.2
   Compiling slab v0.4.7
   Compiling tokio v1.20.1
   Compiling futures-macro v0.3.21
   Compiling tokio-macros v1.8.0
   Compiling futures-executor v0.3.21
   Compiling futures v0.3.21
   Compiling rust-tdlib v0.4.0
error[E0432]: unresolved imports `tokio::task::JoinHandle`, `tokio::time`
  --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\client\worker.rs:25:5
   |
25 |     task::JoinHandle,
   |     ^^^^^^^^^^^^^^^^ no `JoinHandle` in `task`
26 |     time,
   |     ^^^^ no `time` in the root

error[E0433]: failed to resolve: could not find `runtime` in `tokio`
   --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\client\worker.rs:364:34
    |
364 |             let current = tokio::runtime::Handle::try_current().unwrap();
    |                                  ^^^^^^^ could not find `runtime` in `tokio`

error[E0433]: failed to resolve: could not find `SendTimeoutError` in `error`
  --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\errors.rs:75:39
   |
75 |             tokio::sync::mpsc::error::SendTimeoutError::Timeout(_) => SEND_TO_CHANNEL_TIMEOUT,
   |                                       ^^^^^^^^^^^^^^^^ could not find `SendTimeoutError` in `error`

error[E0433]: failed to resolve: could not find `SendTimeoutError` in `error`
  --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\errors.rs:76:39
   |
76 |             tokio::sync::mpsc::error::SendTimeoutError::Closed(_) => CLOSED_CHANNEL_ERROR,
   |                                       ^^^^^^^^^^^^^^^^ could not find `SendTimeoutError` in `error`

error[E0425]: cannot find function `spawn` in crate `tokio`
   --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\client\worker.rs:336:16
    |
336 |         tokio::spawn(async move {
    |                ^^^^^ not found in `tokio`
    |
help: consider importing this function
    |
2   | use std::thread::spawn;
    |
help: if you import `spawn`, refer to it directly
    |
336 -         tokio::spawn(async move {
336 +         spawn(async move {
    | 

error[E0425]: cannot find function `spawn` in crate `tokio`
   --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\client\worker.rs:363:16
    |
363 |         tokio::spawn(async move {
    |                ^^^^^ not found in `tokio`
    |
help: consider importing this function
    |
2   | use std::thread::spawn;
    |
help: if you import `spawn`, refer to it directly
    |
363 -         tokio::spawn(async move {
363 +         spawn(async move {
    | 

error[E0425]: cannot find function `spawn` in crate `tokio`
   --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\client\worker.rs:410:16
    |
410 |         tokio::spawn(async move {
    |                ^^^^^ not found in `tokio`
    |
help: consider importing this function
    |
2   | use std::thread::spawn;
    |
help: if you import `spawn`, refer to it directly
    |
410 -         tokio::spawn(async move {
410 +         spawn(async move {
    | 

error[E0412]: cannot find type `SendTimeoutError` in module `tokio::sync::mpsc::error`
  --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\errors.rs:72:40
   |
72 | impl<T> From<tokio::sync::mpsc::error::SendTimeoutError<T>> for Error {
   |                                        ^^^^^^^^^^^^^^^^ not found in `tokio::sync::mpsc::error`

error[E0412]: cannot find type `SendTimeoutError` in module `tokio::sync::mpsc::error`
  --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\errors.rs:73:44
   |
73 |     fn from(err: tokio::sync::mpsc::error::SendTimeoutError<T>) -> Self {
   |                                            ^^^^^^^^^^^^^^^^ not found in `tokio::sync::mpsc::error`

error[E0207]: the type parameter `T` is not constrained by the impl trait, self type, or predicates
  --> C:\Users\****\.cargo\registry\src\github.com-1ecc6299db9ec823\rust-tdlib-0.4.0\src\errors.rs:72:6
   |
72 | impl<T> From<tokio::sync::mpsc::error::SendTimeoutError<T>> for Error {
   |      ^ unconstrained type parameter

error: aborting due to 10 previous errors

Some errors have detailed explanations: E0207, E0412, E0425, E0432, E0433.
For more information about an error, try `rustc --explain E0207`.
error: could not compile `rust-tdlib` due to 11 previous errors
Process finished with exit code 101
```

**Comment by anwsonwsymous** [comment]
I had this problem too and fixed it with adding this to `Cargo.toml`
```
tokio = {version = "1", features = ["macros", "rt-multi-thread", "sync", "time"]}
```

**Comment by antonio-antuan** [comment]
@Yeyvo thanks for asking the question.
Could you provide your `Cargo.toml` and a command that you use to build the example?

**Comment by antonio-antuan** [comment]
fixed 0.4.3 has just been released
